problems/kattis/R2/Python/tyrado/r2.py

```diff
@@ -0,0 +1,19 @@
+
+def r2():
+    i=(int( input("put in your numbers: ")))
+    j=(int(input("put in number 2: ")))
+    print(i ,j)
+    
+    if i <= 1000 and i>= -1000 and j>=-1000 and j<=1000:
+        k =  0 
+        k=((j*2)-i)
+        print(k);
+        j= (i+k)/2
```

Comment:
j= equation seems unnecessary. Through substitution with the k = equation it simplifies to j = j.

Reply:
I had it in their to make sure I had the k equation right